android/src/main/java/com/bugsnag/BugsnagReactNative.java

```diff
@@ -217,6 +217,14 @@ private void configureRuntimeOptions(Client client, ReadableMap options) {
               client.setReleaseStage(releaseStage);
       }
 
+      if (options.hasKey("autoNotify")) {
+          if (options.getBoolean("autoNotify")) {
+              client.enableExceptionHandler();
+          } else {
+              client.disableExceptionHandler();
+          }
+      }
+
       if (options.hasKey("codeBundleId")) {
           String codeBundleId = options.getString("codeBundleId");
           if (codeBundleId != null && codeBundleId.length() > 0)
```

Comment:
It would be nice to extract the literals in this file into a PSFS constant, as then if our JS API ever changed, we'd only need to edit the value in one place.

Reply:
Yeah. Added a note for this for the backlog.